jupyter-book/linear_models/linear_models_quiz_m4_05.md

```diff
@@ -4,8 +4,8 @@
 By default, a `LogisticRegression` in scikit-learn applies:
 
 - a) no penalty
-- b) a l2 penalty
-- c) a l1 penalty
+- b) a penalty whose strength can be tuned by the user
+- c) a cross-validation to select the penalty strength
```

Comment:
I still would prefer to have a question linked to the penalty here. Indeed, the documentation in scikit-learn provide "none", "l1", "l2" penalty as parameter. So I would propose to:

- add a link to the documentation to make the quiz easier (basically a hyperlink from `LogisticRegression` to the doc)
- add the following solution that is more explicit and linked with the course:

```
- b) shrunk the magnitude of the weights toward zero (also called "l2 penalty")
- c) set some weights exactly to zero (also called "l1 penalty")
```

I think that we wanted people to discover (or make sure that they are aware) that `LogisticRegression` has a default penalty by default.